### ISSUE TITLE
Transmission list method add uri parameters

### ISSUE THREAD

**Issue by ctomatis** [SparkPost/python-sparkpost#133]
https://developers.sparkpost.com/api/transmissions.html#transmissions-list-get
```python
def list(self, **kwargs):
    """
    Get a list of your transmissions....
    
    :param campaign_id: ID of the campaign
    :param template_id: ID of the template
    """
    results = self.request('GET', self.uri, params=kwargs)
    return results
```

**Comment by richleland** [comment]
Hi @ctomatis interested in submitting this as a PR?

**Comment by richleland** [comment]
Resolved by #140
